src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -124,24 +147,26 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
         <div>
           <Spacer y={20} />
           <HorizontalRule />
-          <ToggleRow
-            onToggle={() =>
-              handleScenarioChange({ dailyReports: !scenario?.dailyReports })
-            }
-            toggled={scenario?.dailyReports}
-            label="Subscribe to daily reports"
-            labelHelp="If enabled, your baseline scenario will be shared with Recidiviz and CSG. This data will only be used to provide you with daily reports."
-          />
-          <HorizontalRule />
-          <ToggleRow
-            onToggle={() =>
-              handleScenarioChange({ dataSharing: !scenario?.dataSharing })
-            }
-            toggled={scenario?.dataSharing}
-            label="Share data to improve the model"
-            labelHelp="If enabled, your baseline scenario will be made available to Recidiviz and the research community to improve the model and the state of research on the spread of disease in facilities. Any public research will anonymize state and facility names."
-          />
-          <HorizontalRule />
+          <ToggleContainer hidden={!scenario?.baseline}>
```

Comment:
We only show the toggles and impact button for the baseline scenario.  For now at least